core/subprojects/README.md

```diff
@@ -0,0 +1,21 @@
+# core/subprojects
+
+## What's this? What _is_ this?
+
+- These are subprojects accessible to the `core` meson project.
+
+## icu
+
+### Why are there several files for ICU?
+
+- Ideally we would use meson's [wrapdb](https://mesonbuild.com/Wrapdb-projects.html), and in fact the files here are sourced from that project.
+- However, it requires a much newer meson version to work properly: 0.55 at least.
+- So the file `packagefiles/icu-minimal.zip` is generated from the files in [`packagefiles/icu`](./packagefiles/icu/)
+- Regenerate it with these commands:
```

Comment:
OK, so now that we are looking to use pkgconfig on Linux, we should be able to assume meson 0.55 for other platforms, so we don't need these files?

(There's also the question of which min ver of Debian to support for Keyman 17.0 (@ermshiperete?)

Reply:
That's a question. What's the minimum meson for the non linux builds? I'll then simplify the files accordingly.

Reply:
More specifically:
- at one level we can get rid of the zip file
- at another level we can get rid of most of the files, and only need to override somewhat for the trimming. 

Upstream in wrap db isn't very configurable. I'm planning to contribute some improvements (separately).

Reply:
> What's the minimum meson for the non linux builds?

Min meson version for Keyman builds on Windows is already 1.0 and can be 1.0 on macOS as well. It is only Debian which blocks us here.

(Used to be that IE on Windows was our blocker. Today it is always one of Debian, Safari, or Android. How times have changed.)

Notes:

* mac build docs do not currently state a min meson version; brew.sh has [1.0.1](https://formulae.brew.sh/formula/meson#default) as current version.
* Windows build docs spec 1.0 as min version, and suggest using pip to install as at time of writing, chocolatey only had 0.55. choco now lists [1.0.1](https://community.chocolatey.org/packages/meson) also.

Reply:
I didn't change the minimum because that would break debian. However I changed the wrapfile to a minimum of 0.63.0 (it's what is on my local mac via homebrew) and I know it will function.

Reply:
actually the mac build agent is 0.58.1

Reply:
actually the mac build agent is 0.58.1